assets/src/js/media_archive_editor_view.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable */
+import {createRoot} from '@wordpress/element';
+import ArchivePicker from '../js/Components/ArchivePicker/ArchivePicker';
+
+var frame = wp.media.view.MediaFrame.Select;
+
+const mediaArchiveTab = () => {
+  wp.media.view.MediaFrame.Select = frame.extend({
+    initialize: function() {
+      frame.prototype.initialize.apply(this, arguments);
+
+      var State = wp.media.controller.State.extend({
+        insert: function() {
+          this.frame.close();
+        }
+      });
+
+      this.states.add([
+        new State({
+          id: 'mediaArchive1',
```

Comment:
The `id` name `mediaArchive1` is intentional or we can use just `mediaArchive`? 🤔